src/coreclr/vm/comutilnative.cpp

```diff
@@ -1552,7 +1552,7 @@ extern "C" void QCALLTYPE Interlocked_MemoryBarrierProcessWide()
 static BOOL HasOverriddenMethod(MethodTable* mt, MethodTable* classMT, WORD methodSlot)
 {
     CONTRACTL{
-        NOTHROW;
+        THROWS;
```

Comment:
What has changed that causes this to throw?

Reply:
This calls GetRestoredSlot that THROWS after this change.